src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -628,7 +629,7 @@ protected boolean rescaleGlsk(List<SensitivityFactorGroup<V, E>> factorGroups, S
     protected void writeSkippedFactors(Collection<LfSensitivityFactor<V, E>> lfFactors, SensitivityValueWriter valueWriter) {
         List<LfSensitivityFactor<V, E>> skippedFactors = lfFactors.stream().filter(factor -> factor.getStatus() == LfSensitivityFactor.Status.SKIP).collect(Collectors.toList());
 
-        skippedFactors.forEach(factor -> valueWriter.write(factor.getContext(), null, -1, 0, Double.NaN));
+        skippedFactors.forEach(factor -> valueWriter.write((SensitivityFactor) factor.getContext(), null, -1, 0, Double.NaN));
```

Comment:
Just for my learning: why do you need this cast?

Reply:
Because for some reason AbstractLfSensitivityFactor has a generic non typed Object as context and not explicitly a SensitivityFactor. There may have been a reason for this at some point but I am not sur it is still relevant.

Reply:
In fact SensitivityValueWriter did take a generic object as context parameter before it was moved to the core. This is why the cast was not needed before.